power_grid_model_c/power_grid_model_cpp/include/power_grid_model_cpp/serialization.hpp

```diff
@@ -0,0 +1,84 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#pragma once
+#ifndef POWER_GRID_MODEL_CPP_SERIALIZATION_HPP
+#define POWER_GRID_MODEL_CPP_SERIALIZATION_HPP
+
+#include "power_grid_model_c/serialization.h"
+
+#include "basics.hpp"
+#include "dataset.hpp"
+#include "handle.hpp"
```

Comment:
here and in other locations:
* first include private header files from current library (if any; not here)
* then include public headers from current library
* then include public headers from other libraries in current project
* then include public headers from 3rd party packages (if any; not here)
* then include system headers (if any; not here)
* then include STL headers

```suggestion
#include "basics.hpp"
#include "dataset.hpp"
#include "handle.hpp"

#include "power_grid_model_c/serialization.h"
```